pandas/_libs/window/indexers.pyx

```diff
@@ -74,14 +78,27 @@ def calculate_variable_window_bounds(
     # right endpoint is open
     else:
         end[0] = 0
+    if center_window:
+        for j in range(0, num_values+1):
+            if (index[j] == index[0] + index_growth_sign*window_size/2 and
```

Comment:
use spaces around operators & parens to format this (we don't lint these files automatically)